### ISSUE TITLE
Rufus crash

### ISSUE THREAD

**Issue by stackus** [projectjellyfish/api#831]
Looks like the task Rufus is supposed to execute is missing or incorrectly named.

```
{ 70148382805380 rufus-scheduler intercepted an error:
  70148382805380   job:
  70148382805380     Rufus::Scheduler::EveryJob "15m" {}
  70148382805380   error:
  70148382805380     70148382805380
  70148382805380     RuntimeError
  70148382805380     Don't know how to build task 'upkeep:update_budgets'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task_manager.rb:62:in `[]'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:353:in `[]'
  70148382805380       /Users/stack/source/bah/api/config/initializers/task_scheduler.rb:15:in `block in <top (required)>'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:234:in `call'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:234:in `do_call'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:258:in `do_trigger'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:300:in `block (3 levels) in start_work_thread'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:303:in `call'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:303:in `block (2 levels) in start_work_thread'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:289:in `loop'
  70148382805380       /Users/stack/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rufus-scheduler-3.1.3/lib/rufus/scheduler/jobs.rb:289:in `block in start_work_thread'
```

**Comment by mafernando** [comment]
taking a look at this per Stack's direction. :+1:

**Comment by mafernando** [comment]
Should be addressed by PR #839.
